tests/services/test_schedule_service.py

```
@@ -1,9 +1,12 @@
-import pytest
 from datetime import datetime, timezone
 from unittest.mock import AsyncMock, patch
-from src.services.schedule_service import ScheduleService
-from src.models import User, Schedule
+
+import pytest
 import pytz
+from sqlalchemy.exc import NoResultFound
```

Ruff (0.11.9)

`sqlalchemy.exc.NoResultFound` imported but unused

Remove unused import: `sqlalchemy.exc.NoResultFound`

(F401)